src/publisher.rs

```diff
@@ -10,15 +10,15 @@ use crate::IceoryxError;
 use std::marker::PhantomData;
 use std::mem::MaybeUninit;
 
-pub struct PublisherBuilder<'a, T: ShmSend> {
+pub struct PublisherBuilder<'a, T: ShmSend + ?Sized> {
```

Comment:
is ?Sized accepted in a trait declaration ex. `pub unsafe trait ShmSend: ?Sized {}`, or is it special because it's an anti-constraint?

Reply:
traits are `?Sized` by default and one would have to set `Sized` as trait bounds. But since I want to have `[u8]` as type I cannot do it.